### PR TITLE
i18n: Language empathy mode

### PR DESCRIPTION
**Pull request by akirk**
This is a first sample implementation of what we're calling Language Empathy mode. It allows a user to experience Calypso in a translation state of language X:
- All text that is actually translated will appear in English.
- All text that is not translated, will print "I don't understand".

Here is an example screenshot for Croatian:
image

You could enable it like this:
image

Todo:
- [x] Add Empathy checkbox to language selector
- [x] Make it more obvious that you are in empathy mode.
- [x] Find a better way than just displaying "untranslated", for example rather add styling?

see also D44550-code and https://github.com/Automattic/i18n-issues/issues/27

### CODE REVIEW

**Comment by hypest** [comment]
This is such a cool idea to raise empathy!

**Comment by lancewillett** [comment]
This concept is super cool. 👍 What's needed to move this along?

Six months later—still valid?

For reviews using `Needs Review` label might help get attention, or pinging individuals folks to look.

**Comment by akirk** [comment]
This has been put on the backburner because of other priorities. It would be still a nice thing for the next hack week.

File: client/components/language-picker/modal.jsx [hunk 1]
```diff
@@ -447,6 +449,19 @@ export class LanguagePickerModal extends PureComponent {
 		);
 	}
 
+	renderEmapthModeCheckbox() {
```
Comment (akirk) [inline client/components/language-picker/modal.jsx:452]: Typo: `renderEmpathModeCheckbox`

File: client/lib/i18n-utils/empathy-mode.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+export function enableLanguageEmpatyhMode() {
+	// wrap translations from i18n
+	i18n.registerTranslateHook( ( translation, options ) => {
+		const locale = i18n.getLocaleSlug();
+		if ( 'en' === locale ) {
+			return translation;
+		}
+
+		if ( i18n.hasTranslation( options.original ) ) {
```
Comment (akirk) [inline client/lib/i18n-utils/empathy-mode.js:28]: I guess we need to whitelist the `I don't understand`, no?

**Review by yuliyan — APPROVED** [review]
LGTM  🚀

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3818659

Thank you @akirk for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3818659).

**Pull request merged**